File: bench/multi/sieve.py
n = 1000000; comp = bytearray(n+1); cnt = 0
for i in range(2, n+1):
    if not comp[i]:
        cnt += 1
        for j in range(i*i, n+1, i): comp[j] = 1
print("pi =", cnt)

File: bench/multi/sieve.js
const n = 10000000, comp = new Uint8Array(n+1); let cnt = 0;
for (let i = 2; i <= n; i++) if (!comp[i]) { cnt++; for (let j = i*i; j <= n; j += i) comp[j] = 1; }
console.log("pi =", cnt);
